infrastructure/keymanager/src/mnemonic.rs

```diff
@@ -258,7 +272,7 @@ impl Mnemonic for SecretKey {
 mod test {
```

Comment:
This is quite a big module, perhaps move this to the test folder. This will make the code base smaller for people to check

Its located in: 
```
crate folder/
        src/
        test/    <--- in here
```

Reply:
Personal preference, I'm not overly fussed on this

Reply:
When the file gets bigger, then I will move the tests